build.cmd

```diff
@@ -0,0 +1,16 @@
+call "C:\Apache Software Foundation\apache-maven-3.5.0\bin\mvn" clean package
```

Comment:
I'd prefer to exclude scripts like these, as they are incredibly specific to one person's setup, and of no use to anyone else. Can you move this out please - or possibly add to .gitignore?